pkgs/desktops/gnome-3/default.nix

```diff
@@ -42,6 +42,7 @@ lib.makeScope pkgs.newScope (self: with self; {
   evince = callPackage ./core/evince { }; # ToDo: dbus would prevent compilation, enable tests
 
   evolution-data-server = callPackage ./core/evolution-data-server { };
+  evolution-ews = callPackage ./misc/evolution-ews { };
```

Comment:
Could you move this straight to `pkgs/applications/networking/mailreaders/`. I am slowly disbanding the `gnome3` package set and evolution will be moved to `all-packages.nix` next.

Reply:
I don't quite get what you mean. Do I have to make an `pkgs/applications/networking/mailreaders/evolution` folder an a `evolution-ews.nix` file inside that?

Reply:
I think `pkgs/applications/networking/mailreaders/evolution/evolution-ews/default.nix` makes sense. Especially if we package other extensions in the future.

Reply:
done, thanks for suggestion